tests/test_build.py

```diff
@@ -292,3 +292,50 @@ def test_toc_numbered(
     file_regression.check(
         toc.prettify(), basename=toc_file.split(".")[0], extension=".html"
     )
+
+
+@pytest.mark.parametrize(
+    "toc_file",
+    (
+        "_toc_numbered.yml",  # Numbered at top-level
```

Comment:
Presumably these files are all already tested in the multitoc-numbering extension, no? Are we just duplicating them here? If so, why don't we just have a single test to make sure that the config flag works as expected, and then rely on `multitoc-numbering` to test the actual various TOC styles and such? Or is there some jupyter-book-specific functionality that we should really test here?

Reply:
thanks @choldgraf -- this is really just testing the influence of the `_config.yml` over the resulting build to make sure the `use_multitoc_numbering: false` is working. There are technical tests in `sphinx-multitoc-numbering` to test the `logic`